src/mentors.json

```diff
@@ -769,5 +769,30 @@
         "id": "UGPGRH2RY"
       }
     ]
+  },
+  {
+    "id": "zacharybraddy@gmail.com",
+    "name": "Zac Braddy",
+    "avatar": "https://avatars0.githubusercontent.com/u/794241",
+    "title": "Lead Fullstack Javascript Developer",
+    "description": "I'm a lead developer and budding architect. I love talking about tech!",
+    "country": "United Kingdom",
+    "tags": [
+      "nodejs", "microservices", "kubernetes", "C#", "React"
```

Comment:
We already have `react`. Can you use it please?

Reply:
Sure thing bud. Sorry didn't realise it was an enum. Maybe we could point people towards that list of tags in the code base in the instructions in the readme. Happy to make another PR for that if you'd like?